bpython/debugger.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python2.7
+# -*- coding: utf-8 -*-
+# vim:ts=4:sw=4:softtabstop=4:smarttab:expandtab
+
+"""
+Debugger factory. Set PYTHON_DEBUGGER to a module path that has a post_mortem
+function in it. Defaults to pdb. This allows alternate debuggers to be used,
+such as pycopia.debugger. :)
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+import os
+import sys
+
+
+post_mortem = None
+
+def _get_debugger():
+    global post_mortem
+    modname = os.environ.get("PYTHON_DEBUGGER", "pdb")
```

Comment:
Ok. But that one doesn't always get back to the original bpython, and can also fail if you don't have all the dependencies (which I don't have because the curses variant doesn't need them).